client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+            "sshRSA": "ssh-rsa ...",
+            "saveKeys": "Speichere SSH Schlüssel",
+            "confirmDisableKeys": "\"confirm\" eingeben. Schliess Dich nicht aus!",
+            "disableKeys": "Dauerhaft das hochladen von SSH Schlüssel abschalten",
+            "passwordsMismatch": "Passwörter stimmen nicht überein."
+        },
+        "carpetMode": {
+            "pageTitle": "Teppichmodus Einstellungen",
+            "explanation": "Im Teppichmodus erkennt der Saugroboter Teppiche automatisch und erhöht entsprechend die Saugleistung.",
+            "warning": "Ändern Sie einen der folgenden Parameter nur, wenn Sie wissen, was Sie tun!",
+            "currentLow": "Momentan Niedrig",
+            "currentHigh": "Momentan Hoch",
+            "currentIntegral": "Momentan Integral",
+            "stallTime": "Standzeit",
+            "confirm": "Sollen die im Teppichmodus vorgenommenen Änderungen wirklich gespeichert werden?"
+        },
+        "cleaningHistory": {
+            "header": "Letzte Reinigungsläufe",
+            "loadMore": "Mehr...",
+            "showMap": "Reinigungskarte anzeigen",
+            "historyRecord": "#{{id}} gestartet am {{startDate}}",
+            "duration": "Dauer",
+            "area": "Bereich",
+            "completed": "Vollständig",
+            "code": "Code",
+            "unknownState": "clean_record_map hat unbekannten Zustand zurückgegeben: {{state}}",
+            "mapTitle": "Reinigungskarte von {{cleaningDate}}"
+        },
+        "consumables": {
+            "mainBrush": "Hauptbürste",
+            "sideBrush": "Seitenbürste",
+            "filter": "Filter",
+            "sensorCleaning": "Sensorreinigung",
+            "allStatistics": "Gesamtstatistik",
+            "totalCleanedArea": "Gesamter gereinigter Bereich:",
+            "totalHours": "Gesamte Reinigungsstunden:",
+            "totalCount": "Gesamte Reinigungsanzahl:",
+            "hoursLeft": "{{hours}} Stunde verbleibend",
+            "hoursLeft_plural": "{{hours}} Stunden verbleibend",
+            "confirm": "Soll dieses Verbrauchsmaterial zurückgesetzt werden?"
+        },
+        "info": {
+            "system": "System",
+            "firmwareVersion": "Firmware Version:",
+            "firmwareBuild": "Firmware Build:",
+            "firmwareUpdate": "Firmware Update:",
+            "firmwareUpdateLink": "[send request]",
+            "currentToken": "Aktueller Token",
+            "token": "Token:",
+            "appLocale": "App-Gebietsschema",
+            "appLocaleName": "Name:",
+            "appLocaleBom": "Bom:",
+            "appLocaleLocation": "Ort:",
+            "appLocaleLanguage": "Sprache:",
+            "appLocaleTimezone": "Zeitzone:",
+            "appLocaleLogserver": "Logserver:",
+            "valetudo": "Valetudo",
+            "currentValetudoVersion": "Momentane Valetudo Version:",
+            "newValetudoVersion": "Neue Valetudo Version:",
```

Comment:
should not be much longer than the english text on small screens.
"newValetudoVersion": "Neueste Valetudo-Version:",

Reply:
has been changed to "Letzte aktuelle Valetudo Version"

Reply:
Okay, but be aware that phones like the iPhone SE or the Xperia XZ1/2 compact will cut off "Version" in that case.